components/AccountVerificationForm/AccountVerificationFormProvider.js

```diff
@@ -306,10 +306,6 @@ function useBasiqConnection({ currentStep, userId }) {
       return;
     }
     if (completed) {
```

Comment:
You can remove this if condition block completely, there is no need to leave it as empty @albatahmet